lib/howitzer/web/page.rb

```diff
@@ -129,8 +129,7 @@ def self.expanded_url(params = {})
         if url_template.nil?
           raise PageUrlNotSpecifiedError, "Please specify url for '#{self}' page. Example: url '/home'"
         end
-        host = self == Howitzer::Web::BlankPage ? '' : Helpers.app_url
-        "#{host}#{Addressable::Template.new(url_template).expand(params)}"
+        "#{@root_url unless self == Howitzer::Web::BlankPage}#{Addressable::Template.new(url_template).expand(params)}"
```

Comment:
instead of condition, please specify root_url '' in Howitzer::Web::BlankPage class